### PR TITLE
RESTWS-663 - Fix SwaggerSpecificationCreator#hasSearchHandler(resName)

### PR DESCRIPTION
**Pull request by gayanW**
* Fix invalid logic in SwaggerSpecificationCreator#hasSearchHandler(resourceName)
* Remove undeclared methods
* Remove unnecessary test case SwaggerSpecificationCreatorTest#swaggerSerializeTest()

Issue: [RESTWS-663](https://issues.openmrs.org/browse/RESTWS-663)

### CODE REVIEW

**Comment by dkayiwa** [comment]
@gayanW can you always add a link to the ticket for easier navigation?

**Comment by gayanW** [comment]
Okay. Will do from now on.

**Comment by dkayiwa** [comment]
I thought the ticket was only about Invalid logic in hasSearchHandler(resourceName) and nothing more.

File: omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java [hunk 1]
```diff
@@ -1164,24 +1127,22 @@ private Operation createOperation(DelegatingResourceHandler<?> resourceHandler,
...
 	
-	private String getResourceUrl(String baseUrl, String resourceName) {
-		//Set the root url.
-		return baseUrl + "/v1/" + resourceName;
-	}
-	
-	private boolean hasSearchHandler(String resourceName) {
-		for (SearchHandlerDoc doc : searchHandlerDocs) {
-			if (doc.getResourceURL().contains(resourceName)) {
+	private boolean hasSearchHandler(String resourceName, String resourceParentName) {
```
Comment (gayanW) [inline omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java]: Here goes the actual changes. getParametersListForSearchHandlers(), and getResourceUrl() was unused. So I removed them.

Comment (dkayiwa) [reply omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java]: Do you mind have separate pull requests?

Comment (dkayiwa) [reply omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java]: @gayanW ping 😊

**Pull request merged**